_posts/2022-05-10-oracle-cloud-infrastructure-search-service.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Partner Highlight: Oracle Cloud Infrastructure Search Service with OpenSearch is GA!"
+authors:
+  - andhopp
+  - vachshah
+  - dtaivpp
+date: 2022-05-10
+categories:
+  - partners
+
+excerpt: "Oracle has released it's Cloud Infrastructure Search Service!"
+
+---
+
+# Partner Highlight: Oracle Cloud Infrastructure Search Service with OpenSearch is GA! 
+
+Late last year, we [shared](https://opensearch.org/blog/partners/2021/12/oracle-announcement/) the news that Oracle joined the OpenSearch community and launched the public beta of their new service, OCI Search Service with OpenSearch. Today we’d like to congratulate them on the global availability of their service in all thirty OCI regions. The offering is a fully managed search service based on OpenSearch. The service automates common operational activities like patching, updating, upgrading, resizing, and backups. Using the reference architectures, customers will have access to a validated architecture to deploy and configure the service for log aggregation and in-application search.
+
+We are excited for them as they move from public beta to general availability and even more excited to see their commitment to the OpenSearch community. Of course, you know Oracle and their long legacy in databases and their contribution to the open-source community through Java and MySQL, so it is great to see them actively supporting the OpenSearch project. We expect to see additional open source partnerships from them in the future.
+
+Oracle’s mission is "to help people see data in new ways, discover insights, unlock endless possibilities" and delivering this cloud-based search service with OpenSearch is inline with that mission. Their Gen 2 cloud is a strong foundation with its built-in security to mitigate threats, improved automation, scalability, and high performance.
+
+During the beta customers were able to easily migrate the Elasticsearch infrastructure to OpenSearch. Because their private endpoints use the same APIs as Elasticsearch, it was simply a matter of ingesting data and pointing to the new endpoints. In fact, one of their customers ingested 300 million items into their OpenSearch cluster and is now enabled for analysis and action with OpenSearch Dashboards.
+
+It’s a good day for open source and the OpenSearch community as another hyperscale cloud provider embraces OpenSearch. Congrats again. To learn more about OCI Search Service with OpenSearch go [here](http://oracle.com/cloud/search). 
+
+## Get involved!
+We would love to see you contribute to OpenSearch (if you’re not already)! For almost any type of contribution, the first step is opening an GitHub issue (https://github.com/opensearch-project/OpenSearch). Even if you think you already know what the solution is, writing a description of the problem you’re trying to solve will help everyone get context when they review your pull request. 
```

Comment:
I think we should hyperlink in the [typical](google.com) form. Feels a bit cleaner in most cases.